README.md

```diff
@@ -32,6 +32,7 @@ And the list goes on.
 
 ![Python](https://img.shields.io/badge/python-3670A0?style=for-the-badge&logo=python&logoColor=ffdd54)
 ![SQLite](https://img.shields.io/badge/sqlite-%2307405e.svg?style=for-the-badge&logo=sqlite&logoColor=white)
+![SQLAlchemy](https://img.shields.io/badge/-SQLAlchemy-blue)
```

Comment:
Can you please update the url after `/badge/` to `SQLAlchemy-blue.svg?logoColor=white&style=for-the-badge&color=red`

This is to maintain this badge's style consistent with other badges in the README file.

Reply:
ok @Aditya-Gupta1